### PR TITLE
Update DCR Ad styles (Carrot, Fluid)

### PR DESCRIPTION
**Pull request by mxdvl**
## What does this change?

Removes inline ad render style for carrot ads.

## Does this change need to be reproduced in dotcom-rendering ?

- [ ] No
- [X] ~~Yes (quickfix) https://github.com/guardian/dotcom-rendering/pull/3060~~
- [X] Yes https://github.com/guardian/dotcom-rendering/pull/3064

## What is the value of this and can you measure success?

Idiomatic code.  
A proper implementation of #22768.

### Tested

- [x] Locally
- [x] On CODE (optional)

### CODE REVIEW

File: static/src/javascripts/projects/commercial/modules/carrot-traffic-driver.js [hunk 1]
```diff
@@ -96,47 +74,18 @@ const desktopRules = {
...
-        ' .element-pullquote': {
-            minAbove: 400,
-            minBelow: 400,
-        },
     },
-    fromBottom: true,
 };
 
 // mobile(320) and above
 const mobileRules = {
```
Comment (arelra) [inline static/src/javascripts/projects/commercial/modules/carrot-traffic-driver.js]: It looks like `desktopRules ` and `mobileRules` are equivalent objects with the same props and values?

**Review by arelra — COMMENTED** [review]
_If_ `desktopRules ` and `mobileRules` are equivalent and the `default` case of `getRules` is a failure case, then potentially we are applying the same rules in every case and we don't need the different rules or the current complexity of the switch statement.

File: static/src/javascripts/projects/commercial/modules/carrot-traffic-driver.js (hunk 1 above)
Comment (mxdvl) [reply static/src/javascripts/projects/commercial/modules/carrot-traffic-driver.js]: Good shout, [they are indeed identical](https://editor.mergely.com/E2ygyVc3/).

**Comment by mxdvl** [comment]
@arelra thanks for the insightful comment. I was trying to simplify the rules from the very verbose original, but there’s room for even even more compression.

See the diff between `default` and `desktop`/`mobile` rules [here](https://editor.mergely.com/FZHnf479/).

**Comment by arelra** [comment]
> @arelra thanks for the insightful comment. I was trying to simplify the rules from the very verbose original, but there’s room for even even more compression.
> 
> See the diff between `default` and `desktop`/`mobile` rules [here](https://editor.mergely.com/FZHnf479/).

You are welcome :)
Very minor so up to you if you're refactoring - I wonder if `default` naming is slightly misleading as this is the exceptional case and the default is actually the desktop/mobile case.

**Review by arelra — APPROVED** [review]
🆒

**Pull request merged**